<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1282</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check Deflection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../civiltools.qrc">
    <normaloff>:/civiltools/images/deflection.svg</normaloff>:/civiltools/images/deflection.svg</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Dead</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QComboBox" name="dead_combobox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="sdead_combobox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="partition_combobox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Dead</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Supper-Dead</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Partition</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Lives</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="4" column="1">
           <widget class="QComboBox" name="live5_combobox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>L-RED</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>Live_0.5</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="lred_combobox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Live</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="live_combobox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_33">
            <property name="text">
             <string>Parking</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="live_parking_combobox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>SNOW</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="lroof_combobox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="snow_combobox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>LROOF</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Live Load Percentage</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QDoubleSpinBox" name="live_percentage_spinbox">
            <property name="suffix">
             <string/>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
            <property name="value">
             <double>0.250000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Filename:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filename">
           <property name="text">
            <string>BeamDeflections</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="check_button">
           <property name="text">
            <string>Check</string>
           </property>
           <property name="icon">
            <iconset resource="../../civiltools.qrc">
             <normaloff>:/civiltools/images/tick.svg</normaloff>:/civiltools/images/tick.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="create_report_button">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Report</string>
           </property>
           <property name="icon">
            <iconset resource="../../civiltools.qrc">
             <normaloff>:/civiltools/images/word.svg</normaloff>:/civiltools/images/word.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="cancel_button">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Close</string>
           </property>
           <property name="icon">
            <iconset resource="../../civiltools.qrc">
             <normaloff>:/civiltools/images/cancel.svg</normaloff>:/civiltools/images/cancel.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="help">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Help</string>
           </property>
           <property name="icon">
            <iconset resource="../../civiltools.qrc">
             <normaloff>:/civiltools/images/help.png</normaloff>:/civiltools/images/help.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="table_vertical_layout">
       <item>
        <widget class="QGraphicsView" name="graphicsview">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="renderHints">
          <set>QPainter::Antialiasing</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="results_vertical_layout">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Check</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="1">
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>Continues Beams</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>Short Term</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_34">
            <property name="text">
             <string>Long Term</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_35">
            <property name="text">
             <string>Console Beams</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="Gui::PrefComboBox" name="continues_short_term_combobox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <property name="prefEntry" stdset="0">
             <cstring>continues_short_term_combobox</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/civilTools</cstring>
            </property>
            <item>
             <property name="text">
              <string>Ln / 180</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ln / 360</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="Gui::PrefComboBox" name="continues_long_term_combobox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="prefEntry" stdset="0">
             <cstring>continues_long_term_combobox</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/civilTools</cstring>
            </property>
            <item>
             <property name="text">
              <string>Ln / 240</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ln / 480</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="Gui::PrefComboBox" name="console_long_term_combobox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="prefEntry" stdset="0">
             <cstring>console_long_term_combobox</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/civilTools</cstring>
            </property>
            <item>
             <property name="text">
              <string>Ln / 120</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ln / 240</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ln / 480</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="Gui::PrefComboBox" name="console_short_term_combobox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="prefEntry" stdset="0">
             <cstring>console_short_term_combobox</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/civilTools</cstring>
            </property>
            <item>
             <property name="text">
              <string>Ln / 180</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ln / 360</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="check_results">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="results">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::PrefComboBox</class>
   <extends>QComboBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dead_combobox</tabstop>
  <tabstop>sdead_combobox</tabstop>
  <tabstop>partition_combobox</tabstop>
  <tabstop>live_combobox</tabstop>
  <tabstop>lred_combobox</tabstop>
  <tabstop>live5_combobox</tabstop>
 </tabstops>
 <resources>
  <include location="../../civiltools.qrc"/>
 </resources>
 <connections/>
</ui>
